bundles/org.openhab.binding.ntp/src/main/java/org/openhab/binding/ntp/internal/discovery/NtpDiscovery.java

```diff
@@ -44,12 +48,12 @@ public NtpDiscovery() throws IllegalArgumentException {
     }
 
     @Override
-    protected void activate(Map<String, Object> configProperties) {
+    protected void activate(@Nullable Map<String, @Nullable Object> configProperties) {
```

Comment:
Why is this needed at all? It only calls the method it overrides

Reply:
I should refactor this call and rather use injection.

Reply:
Constructor injection done.